sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/models/CreateChatThreadOptions.java

```diff
@@ -66,34 +68,44 @@ public CreateChatThreadOptions setParticipants(List<ChatParticipant> participant
     }
 
     /**
-     * Get the repeatabilityRequestID property
+     * Adds another participant to the list of participants to create the chat thread with
+     *
+     * @param participant The participant to add
+     * @return the CreateChatThreadOptions object itself
+     */
+    public CreateChatThreadOptions addParticipant(ChatParticipant participant) {
```

Comment:
Not sure about the need for this method, was it requested by the Java architects? Devs already have access to setParticipants, and this method is just a glorified wrapper for List.add(). 

And can be confused with the actual chatClient.addParticipant()

It also opens us up to a potential NullReferenceException if used like in this corner scenario:
- client wants to create a thread with just the initiator and uses setParticpants(null)
- then reuses the CreateChatThreadOptions object with .addParticipant() to add another participant, which will throw a null reference exception

Reply:
this was explicitly requested in the Azure review.